graphql/client.js

```diff
@@ -13,11 +13,9 @@ const client = new ApolloClient({
 })
 
 export default async function (AEMQuery) {
-  return client
-    .query({
-      query: AEMQuery,
-    })
-    .catch((error) => {
-      console.log(error)
-    })
+  return client.query({
```

Comment:
Should we have a `catch()` here? Good opportunity for our logger as well.

Reply:
I'd rather not catch the error here since we won't know what page is throwing it. I'll add at the page level